Specs/createPackableSpecs.js

```diff
@@ -1,12 +1,21 @@
 /*global define*/
 define([
         'Core/defaultValue',
-        'Core/defined'
+        'Core/defined',
+        'Core/Math'
     ], function(
         defaultValue,
-        defined) {
+        defined,
+        CesiumMath) {
     'use strict';
 
+    function comparePackedArrays(array1, array2) {
```

Comment:
Should this go in [addDefaultMatchers.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/addDefaultMatchers.js)?